mobile-app/app/screens/AppNavigator/screens/Settings/screens/AboutScreen.tsx

```diff
@@ -157,39 +223,45 @@ export function AboutScreen (): JSX.Element {
         ))
       }
 
-      <View style={tailwind('flex-row justify-center items-center text-center mt-2 mb-6')}>
+      <ThemedTextV2
+        dark={tailwind('text-mono-dark-v2-500')}
+        light={tailwind('text-mono-light-v2-500')}
+        style={tailwind('font-normal-v2 text-xs uppercase mt-6 mx-10')}
+      >
+        {translate('screens/AboutSCreen', 'Other information')}
+      </ThemedTextV2>
+      <ThemedViewV2
+        dark={tailwind('bg-mono-dark-v2-00')}
+        light={tailwind('bg-mono-light-v2-00')}
+        style={tailwind('flex-col rounded-lg mx-5 mt-2')}
+      >
         {
-          ABOUT_LINKS.map((l) => (
-            <TouchableOpacity
-              testID={l.testID}
-              style={tailwind('flex-1 p-2 justify-center items-center text-center')}
-              onPress={async () => await openURL(l.url)} key={l.testID}
-            >
-              <ThemedText
-                style={tailwind('text-sm font-medium')}
-              >{translate('screens/AboutScreen', l.title)}
-              </ThemedText>
-            </TouchableOpacity>
+          ABOUT_LINKS.map((link, index) => (
+            <AboutLinkItem
+              link={link}
+              key={link.testID}
+              border={index < ABOUT_LINKS.length - 1}
+            />
           ))
         }
-      </View>
+      </ThemedViewV2>
 
-      <View style={tailwind('items-center mt-2 mb-6')}>
-        <ThemedText
-          dark={tailwind('text-gray-600')} light={tailwind('text-gray-400')}
-          style={tailwind('text-xs font-normal text-center')}
+      <View style={tailwind('items-center my-12 mx-10')}>
+        <ThemedTextV2
+          dark={tailwind('text-mono-dark-v2-500')} light={tailwind('text-mono-light-v2-500')}
+          style={tailwind('font-normal-v2 text-xs font-normal text-center')}
         >
           {translate('screens/CommunityScreen', 'DeFiChain is a community-driven and open project.')}
-        </ThemedText>
+        </ThemedTextV2>
 
-        <ThemedText
-          dark={tailwind('text-gray-600')} light={tailwind('text-gray-400')}
-          style={tailwind('text-xs font-normal text-center')}
+        <ThemedTextV2
```

Comment:
I think this one uses a different font? Maybe because it has font-normal-v2 first, then font-normal?

Reply:
Done refactoring font and layout